helm/hpcc/templates/dali.yaml

```diff
@@ -37,11 +37,13 @@ spec:
   selector:
     matchLabels:
       run: {{ $dali.name | quote }}
```

Comment:
I used a server: label for associating services with pods, rather than a run: label - they don't always match (e.g. sasha services that are done inside the dali, and roxie clusters where not every node is a server).

I left the run: label in place as that seems to be something of a helm convention, and has its uses for interrogating the state of the system via kubectl etc. I don't think we actually use it ourselves now though.